.github/workflows/publish-docker-release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          file: scripts/docker/release.Dockerfile
+          file: scripts/docker/polkadot_builder_docker.Dockerfile
```

Comment:
I am not sure about the `_docker` part... A dockerfile is obviously a docker thing.
IMO that should be: `polkadot_injected_release.Dockerfile`

Reply:
Here I was a little bit confused with naming because from what I understood this file is needed for creating a docker image and publishing it to docker hub. That's why I added `_docker` here as an indicator.

Reply:
IMO the name of the file should describe what the file **IS** and not what it is used for.
For instance, one Dockerfile maybe used in CI and no longer tomorrow (or the other way around) so `ci` does not have a place in the name of an image. So the main question is **what is the image about**.

I would not even mention **apt** in the name as this is an implementation detail. Today we may use apt and tomorrow we will do something else (potentially...).

NOTE: If some images are very similar, they may be generated from the same dockerfile, published with the same name but use another TAG. We may consider this for instance for some debug images with more flags.

Reply:
Sounds good. In this case I'll add info about docker into image description.